server/sonar-server/src/main/java/org/sonar/server/component/es/ProjectMeasuresIndexDefinition.java

```diff
@@ -34,6 +36,12 @@
   public static final String FIELD_MEASURES_KEY = "key";
   public static final String FIELD_MEASURES_VALUE = "value";
 
+  public static final String TYPE_AUTHORIZATION = "authorization";
+  public static final String FIELD_AUTHORIZATION_PROJECT_UUID = "project";
+  public static final String FIELD_AUTHORIZATION_GROUPS = "groups";
```

Comment:
We know that this field should contains group ids. It's not possible yet because of "anyone" and organizations. Still I recommend to rename the field to "groupNames" to remove ambiguity and potential bugs.